sos/plugins/__init__.py

```diff
@@ -209,20 +221,20 @@ def do_cmd_output_sub(self, cmd, regexp, subst):
                     self._log_debug("applying substitution to '%s'" % path)
                     readable = self.archive.open_file(path)
                     result, replacements = re.subn(
-                        regexp, subst, readable.read())
+                        regexp, subst, readable.read(), flags=flags)
                     if replacements:
                         self.archive.add_string(result, path)
-
         except Exception as e:
             msg = "regex substitution failed for '%s' with: '%s'"
             self._log_error(msg % (called['exe'], e))
             replacements = None
         return replacements
 
-    def do_file_sub(self, srcpath, regexp, subst):
+    def do_file_sub(self, srcpath, regexp, subst, ignore_case=False):
```

Comment:
Ditto for this change - we've never needed case-insensitive regex comparison before, why does Kubernetes need it?